### PR TITLE
Add some tools and client libraries

### PR DESCRIPTION
**Pull request by Kidswiss**
Some more suggestions how to use the S3 bucket.

### CODE REVIEW

File: docs/modules/ROOT/pages/object-storage/usage.adoc [hunk 1]
```diff
@@ -1,5 +1,7 @@
 = How to Use an AppCat Object Bucket
 
+== Usage Example within APPUiO
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/usage.adoc:3]:
```suggestion
== Usage Example within an APPUiO Cloud Zone
```

File: docs/modules/ROOT/pages/object-storage/usage.adoc [hunk 2]
```diff
@@ -40,3 +42,16 @@ spec:
 ----
 <1> Values copied from the secret's `ENDPOINT` and `BUCKET` fields.
 <2> Directly referencing the fields in the generated secret `objectbucket-creds`.
+
+== List of Clients and Libraries
+
+If you want to inspect the object buckets or access it outside of APPUiO you can use one of these client tools:
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/usage.adoc:48]:
```suggestion
If you want to inspect the object buckets or access it outside of APPUiO you can use one of these client tools:

```

File: docs/modules/ROOT/pages/object-storage/usage.adoc [hunk 3]
```diff
@@ -40,3 +42,16 @@ spec:
...
+* https://cyberduck.io/
+* https://rclone.org/
+* https://s3tools.org/s3cmd
+* https://filezillapro.com/docs/v3/cloud/configure-filezilla-pro-to-connect-to-s3/
+
+For client libraries to be used in your project:
+go: https://minio-go.min.io/
+javascript: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html
+python: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html
```
Comment (tobru) [inline docs/modules/ROOT/pages/object-storage/usage.adoc:57]:
```suggestion
The following client libraries are a suggestion:

* Go: https://minio-go.min.io/[MinIO Go Client SDK]
* JavaScript: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html[AWS SDK for JavaScript v3]
* Python: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html[Boto3]
```

**Review by tobru — CHANGES_REQUESTED** [review]
For the new section, I suggest adding some one-liners to get the connection information out ouf the generated secret and directly set the well-known environment variables, because I'm pretty sure that this is not clear to anyone how to do that.

**Comment by Kidswiss** [comment]
> For the new section, I suggest adding some one-liners to get the connection information out ouf the generated secret and directly set the well-known environment variables, because I'm pretty sure that this is not clear to anyone how to do that.

Though quite a few of the tools listed don't use those env vars. Except maybe s3cmd.

**Review by tobru — APPROVED** [review]

**Pull request merged**